backup-restore/recipes/edb-backup-restore.yaml

```diff
@@ -1,3 +1,4 @@
+# This recipe has been tested and verified for compatibility with EDB Postgres versions 15 and 16.
```

Comment:
change to - This recipe has been verified against EDB Postgres versions 15 and 16.